bitnami/redis/values.yaml

```diff
@@ -902,6 +910,13 @@ sentinel:
   ## @param sentinel.extraEnvVarsSecret Name of existing Secret containing extra env vars for Redis&trade; Sentinel nodes
   ##
   extraEnvVarsSecret: ""
+  ## @param sentinel.externalMaster.enabled Use external master for bootstrapping
+  ## @param sentinel.externalMaster.host External master host to bootstrap from
+  ## @param sentinel.externalMaster.port Port for external master host
+  externalMaster:
+    enabled: false
+    host: ""
+    port: 26379
```

Comment:
This is the redis port, not the sentinel port. It's a bit confusing as it is the redis port we're asking sentinel to monitor.

```suggestion
    port: 6379
```

Reply:
This makes sense.  I've adjusted this and also modified the description a tad to make this a wee bit more clear.  I'm not seeing issues in testing this with my local setup at the moment.  I think we may be ready to remove the WIP and ping the Maintainers for review.

I did need to perform a rebase, a look behind me would be wise as I did need to resolve a conflict.